documentation/domains/Domain.json

```diff
@@ -184,6 +184,38 @@
         }
       }
     },
+    "CommonMount": {
+      "type": "object",
+      "properties": {
+        "sizeLimit": {
+          "description": "The emptyDir volume size limit. Defaults to unset.",
+          "type": "string"
+        },
+        "mountPath": {
+          "description": "The common mount path. Defaults to /common.",
+          "type": "string"
+        },
+        "targetPath": {
+          "description": "The target mount path. Defaults to \u0027/tmpCommonMount\u0027.",
+          "type": "string"
+        },
+        "containers": {
+          "description": "The common mount containers.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Container"
+          }
+        },
+        "medium": {
+          "description": "The emptyDir volume medium. Defaults to unset.",
+          "type": "string"
+        },
+        "emptyDirVolumeName": {
```

Comment:
Does the volume name need to be configurable? I thought it just needs to be sufficiently unique.

Reply:
It's set to the value `operator-common-volume` that we decided on in the meeting. I thought of leaving it configurable for increased flexibility. Do you foresee any issues due to this?

Reply:
Yes. The more configuration, the more complicated the feature looks to the end user, and the more chance they make a mistake when tuning the configuration. If we choose a 'safe' unique hard coded value, then there'd be no such worries. We use hard coded values in several other similar paths in the Operator for the same reasons.